.gitignore

```diff
@@ -5,3 +5,4 @@
 /views/node_modules
 !vendor/index.php
 !vendor/.htaccess
+views/yarn.lock
```

Comment:
Because we are not using yarn but npm only, it's not necessary to add it and I discourage you to use yarn because of the package-lock file
```suggestion
```